packages/openapi-typescript-helpers/index.d.ts

```diff
@@ -7,6 +7,22 @@ export type OkStatus = 200 | 201 | 202 | 203 | 204 | 206 | 207 | "2XX";
 // biome-ignore format: keep on one line
 export type ErrorStatus = 500 | 501 | 502 | 503 | 504 | 505 | 506 | 507 | 508 | 510 | 511 | '5XX' | 400 | 401 | 402 | 403 | 404 | 405 | 406 | 407 | 408 | 409 | 410 | 411 | 412 | 413 | 414 | 415 | 416 | 417 | 418 | 420 | 421 | 422 | 423 | 424 | 425 | 426 | 427 | 428 | 429 | 430 | 431 | 444 | 450 | 451 | 497 | 498 | 499 | '4XX' | "default";
 
+/**
+ * 'default' returns every not explicitly defined status code
+ * '2XX' returns all 2XX status codes
+ * '4XX' returns all 4XX status codes
+ * '5XX' returns all 5XX status codes
```

Comment:
Shouldn't we remove the explicitly defined status codes from the wildcards? From the OpenAPI Spec (https://swagger.io/specification/#responses-object)

> If a response is defined using an explicit code, the explicit code definition takes precedence over the range definition for that code.

So for example, if we have: "2xx" -> `A` and "204" -> `B`, with different, "204" should resolve to `B` and I think currently it would resolve to `A | B`.